contracts/Vote.sol

```diff
@@ -333,6 +343,46 @@ contract Vote is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed {
         return stCelo;
     }
 
+    /**
+     * Deletes timestamp of expired proposal from storage.
+     * @param proposalId The proposal Id.
+     */
+    function deleteExpiredProposalTimestamp(uint256 proposalId) external {
```

Comment:
if I understand correctly, `deleteExpiredProposalTimestamp()` needs to be called before `deleteExpiredVoterProposalId()` otherwise `deleteExpiredVoterProposalId()`  will revert.

Correct?

If that’s the case, could we call `deleteExpiredProposalTimestamp()` inside `deleteExpiredVoterProposalId()` ?

Reply:
This function could revert with `ProposalNotExpired()` if not expired.

Reply:
Updated